src/front/src/front/component.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+def generate_component(request, component, default_css=True, default_js=True):
+    context = {}
+    if default_css:
+        context["css"] = f"{component}/{component}.css"
+    if default_js:
+        context["js"] = f"{component}/{component}.js"
+    response = render(request, f"{component}/{component}.html", context=context)
+    # TODO: fix it properly
```

Comment:
let's not push todo's in production

Reply:
Thanks, I hadn't seen that error. This code is no longer needed

https://github.com/tdameros/42-transcendence/pull/23/commits/4b331031f15e151c0a04be7866df6e9df9bc07b6